gson/src/main/java/com/google/gson/JsonValue.java

```diff
@@ -0,0 +1,51 @@
+package com.google.gson;
+
+import com.google.gson.stream.JsonWriter;
+
+/**
+ * A class representing a raw JSON value. This should be used with
+ * {@link JsonWriter#jsonValue(String)} and should not be encoded or escaped further.
```

Comment:
> This should be used with

Might be a little bit irritating because the user is not using it; the internal implementation is using it.